website/static/js/osfHelpers.js

```diff
@@ -23,6 +23,35 @@ var growl = function(title, message, type) {
     new GrowlBox(title, message, type || 'danger');
 };
 
+
+/**
+ * Generate OSF absolute URLs, including prefix and arguments. Assumes access to mako globals for pieces of URL.
+ * Can optionally pass in an object with params (name:value) to be appended to URL. Calling as:
+ *   apiV2Url("users/4urxt/applications", {"a":1, "filter[fullname]":"lawrence"}, "https://staging2.osf.io/api/v2/")
+ * would yield the result:
+ *  "https://staging2.osf.io/api/v2/users/4urxt/applications?a=1&filter%5Bfullname%5D=lawrence"
+ * @param {String} pathString The string to be appended to the absolute base path, eg "users/4urxt"
+ * @param {Object} paramsObject (optional) An object containing parameters to add to the URL. Otherwise pass 'undefined'.
+ * @param {String} apiPrefix (optional) Manually specify the prefix used for API routes (useful for testing)
+ */
+var apiV2Url = function (pathString, paramsObject, apiPrefix){
+    apiPrefix = apiPrefix || window.contextVars.apiV2Prefix;
```

Comment:
Sure. The token isn't modified: https://github.com/sloria/webtest-plus/blob/develop/webtest_plus/app.py#L29-L31